src/module/hooks/preCreateChatMessageHandler.mjs

```diff
@@ -125,8 +125,16 @@ export default class preCreateChatMessageHandler {
 
         // In 2e sorcerer breath spells add the E.D. to their crit range
         let addEdToCritRange = false;
+        let addStokeToCritRange = false;
         if (game.settings.get("archmage", "secondEdition")) {
-          addEdToCritRange = options.item.system.breathWeapon?.value?.length > 0;
+            addEdToCritRange = options.item.system.breathWeapon?.value?.length > 0;
+            // Dragons also can increase their crit range.
+            if (options.actor.system.details?.type?.value === 'dragon') {

```

Comment:
Suggestion: shouldn't this be contingent on the stoke resource being active rather than the monster type? Consider some weird dragon with no breath (like at least one type in SotBS) or an undead/dragon monster such as the last fight in The Archmage's Orrery for examples where this trigger might not fire correctly.

Reply:
Good catch. I was implementing this based on the rules text that suggested all dragons get stoke, but making it a toggleable resource and presetting it on the compendium dragons will be better.